src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/DisplayOverviewMap/DisplayOverviewMap.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.DisplayOverviewMap.DisplayOverviewMap"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <esri:MapView x:Name="MyMapView"
```

Comment:
While this layout is completely reasonable, I think for the purposes of demonstrating the ease of use of Toolkit components, it would be best to simplify.

In the case of OverviewMap, there are good defaults for size, vertical alignment, and height. Because of that, all of the row definitions and row settings can be removed. I think for Xamarin.Forms specifically a margin might be needed, but there should be good defaults for the other platforms.